SystemTest/main.go

```
@@ -1,48 +1,241 @@
+	open, err := sql.Open("postgres", "postgres://postgres:postgres@0.0.0.0:5432/test_db?sslmode=disable")
+	if err != nil {
+		panic(err)
+	}
+	defer open.Close()
+	_, err = open.Exec("INSERT INTO public.user_user(password, last_login, is_superuser, username, first_name, last_name, email, is_staff, is_active, date_joined, national_id, phone, address, is_email_verified, verification_code, verification_code_expires_at, sso_user_id, is_banned)\n  VALUES ($1, null, false, $2, '','', $2, false, true, NOW() , null, '09373668000', null, true, null, null, 1, false)", databasePassword, testUserEmail)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func (s *ServiceClient) getProductItem(itemId int) (*Item, error) {
```

golangci-lint (1.62.2)

func `(*ServiceClient).getProductItem` is unused

(unused)